cwltool/validate_js.py

```diff
@@ -25,6 +25,17 @@ def is_expression(tool, schema):
     # type: (Union[CommentedMap, Any], avro.schema.Schema) -> bool
     return isinstance(schema, avro.schema.EnumSchema) and schema.name == "Expression" and isinstance(tool, (str, Text))
 
+class SuppressLog(logging.Filter):
+    def __init__(self, name):  # type: (Text, logging.Logger) -> None
```

Comment:
Just `type: (Text) -> None`?

Reply:
Whoops, yes, I copied it from somewhere else.